src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -5,7 +5,8 @@ import io.circe.generic.auto._
 import sangria.macros.derive._
 import sangria.marshalling.circe._
 import sangria.relay._
-import sangria.schema.{Argument, BooleanType, Field, InputObjectType, IntType, ListType, ObjectType, OptionInputType, OptionType, ProjectedName, Projector, StringType, fields}
+import sangria.schema.{Argument, BooleanType, Context, Field, InputObjectType, IntType, ListType, ObjectType, OptionInputType, OptionType, Projector, StringType, fields}
+import uk.gov.nationalarchives.tdr.api.auth.ValidateHasConsignmentsAccess.reportingRole
```

Comment:
Think this import is unused now: `uk.gov.nationalarchives.tdr.api.auth.ValidateHasConsignmentsAccess.reportingRole`